04_numeronym/luliangLi/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func transfer(in string) string {
+	if len(in) > 3 {
+		head := in[0:1]
+		tail := in[len(in)-1:]
+		body := strconv.Itoa(len(in) - 2)
+
+		return head + body + tail
```

Comment:
Please consider the case whereby the input isn't a word (only containing letters), should that be a numeronym? e.g. "123456789", should the output be "179"? [https://en.wikipedia.org/wiki/Numeronym](url) Wiki pages might give you some clue as to what should be a numeroym.